src/index.html

```diff
@@ -8,9 +8,36 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div>
+      <form action="#"
+       method="post"
+       name="form"
+       data-qa="big"
+       class="searchBar"
+       >
```

Comment:
Fix code style everywhere
```suggestion
      <form
        action="#"
        method="post"
        name="form"
        data-qa="big"
        class="searchBar"
       >
```

Reply:
The same still not fixed